rfcs/checkbox-component.md

```diff
@@ -2,72 +2,74 @@
 
 ### Exports
 
-* `StatefulCheckbox`
-* `StatefulCheckboxContainer`
-* `Checkbox`
-* `StyledRoot`
-* `StyledCheckmark`
-* `StyledLabel`
-* `StyledInput`
+- `StatefulCheckbox`
```

Comment:
what's the reason for changing it from `*` to `-`?

Reply:
was fixed by lint

Reply:
do we lint markdown files? it seems that all other RFCs follow the `*` syntax